<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DLArt_GUI</class>
 <widget class="QMainWindow" name="DLArt_GUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1385</width>
    <height>904</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Deep Learning Art</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>1361</width>
      <height>811</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="Tab_Multiclass">
     <attribute name="title">
      <string>Multiclass Classification</string>
     </attribute>
     <widget class="QGroupBox" name="groupBox">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>511</width>
        <height>431</height>
       </rect>
      </property>
      <property name="title">
       <string>Database and Patching</string>
      </property>
      <widget class="QSpinBox" name="SpinBox_PatchX">
       <property name="geometry">
        <rect>
         <x>90</x>
         <y>241</y>
         <width>60</width>
         <height>31</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="maximum">
        <number>500</number>
       </property>
       <property name="value">
        <number>40</number>
       </property>
      </widget>
      <widget class="QLabel" name="label_5">
       <property name="geometry">
        <rect>
         <x>160</x>
         <y>240</y>
         <width>16</width>
         <height>31</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="text">
        <string>x</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_4">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>240</y>
         <width>71</width>
         <height>31</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>8</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Patch Size:</string>
       </property>
      </widget>
      <widget class="QSpinBox" name="SpinBox_PatchY">
       <property name="geometry">
        <rect>
         <x>180</x>
         <y>240</y>
         <width>60</width>
         <height>31</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="maximum">
        <number>500</number>
       </property>
       <property name="value">
        <number>40</number>
       </property>
      </widget>
      <widget class="QLabel" name="label_6">
       <property name="geometry">
        <rect>
         <x>340</x>
         <y>240</y>
         <width>61</width>
         <height>31</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>8</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Overlap:</string>
       </property>
      </widget>
      <widget class="QDoubleSpinBox" name="SpinBox_PatchOverlapp">
       <property name="geometry">
        <rect>
         <x>400</x>
         <y>240</y>
         <width>71</width>
         <height>31</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="maximum">
        <double>1.000000000000000</double>
       </property>
       <property name="singleStep">
        <double>0.050000000000000</double>
       </property>
       <property name="value">
        <double>0.500000000000000</double>
       </property>
      </widget>
      <widget class="QPushButton" name="Button_Patching">
       <property name="geometry">
        <rect>
         <x>410</x>
         <y>360</y>
         <width>91</width>
         <height>31</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>8</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Patching</string>
       </property>
      </widget>
      <widget class="QTreeWidget" name="TreeWidget_Patients">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>78</y>
         <width>151</width>
         <height>151</height>
        </rect>
       </property>
       <column>
        <property name="text">
         <string notr="true">1</string>
        </property>
       </column>
      </widget>
      <widget class="QTreeWidget" name="TreeWidget_Datasets">
       <property name="geometry">
        <rect>
         <x>180</x>
         <y>78</y>
         <width>321</width>
         <height>151</height>
        </rect>
       </property>
       <column>
        <property name="text">
         <string notr="true">1</string>
        </property>
       </column>
      </widget>
      <widget class="QPushButton" name="Button_DB">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>30</y>
         <width>111</width>
         <height>31</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>8</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Select Database</string>
       </property>
      </widget>
      <widget class="QLabel" name="Label_DB">
       <property name="geometry">
        <rect>
         <x>130</x>
         <y>35</y>
         <width>361</width>
         <height>21</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>8</pointsize>
        </font>
       </property>
       <property name="text">
        <string>No Database selected</string>
       </property>
      </widget>
      <widget class="QPushButton" name="Button_OutputPathPatching">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>360</y>
         <width>91</width>
         <height>31</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>8</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Output Path</string>
       </property>
      </widget>
      <widget class="QLabel" name="Label_OutputPathPatching">
       <property name="geometry">
        <rect>
         <x>120</x>
         <y>360</y>
         <width>271</width>
         <height>31</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>8</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Output Path</string>
       </property>
      </widget>
      <widget class="QPushButton" name="Button_MarkingsPath">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>320</y>
         <width>91</width>
         <height>31</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>8</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Markings Path</string>
       </property>
      </widget>
      <widget class="QLabel" name="Label_MarkingsPath">
       <property name="geometry">
        <rect>
         <x>120</x>
         <y>320</y>
         <width>281</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>Markings Path</string>
       </property>
      </widget>
      <widget class="QRadioButton" name="RadioButton_MaskLabeling">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>290</y>
         <width>121</width>
         <height>17</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>8</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Mask Labeling</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QRadioButton" name="RadioButton_PatchLabeling">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>290</y>
         <width>101</width>
         <height>17</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>8</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Patch Labeling</string>
       </property>
      </widget>
      <widget class="QComboBox" name="ComboBox_Patching">
       <property name="geometry">
        <rect>
         <x>270</x>
         <y>282</y>
         <width>231</width>
         <height>31</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>8</pointsize>
        </font>
       </property>
       <item>
        <property name="text">
         <string>Select Patching Mode ...</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>2D</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>3D</string>
        </property>
       </item>
      </widget>
      <widget class="QSpinBox" name="SpinBox_PatchZ">
       <property name="geometry">
        <rect>
         <x>270</x>
         <y>240</y>
         <width>51</width>
         <height>31</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="value">
        <number>5</number>
       </property>
      </widget>
      <widget class="QLabel" name="label_7">
       <property name="geometry">
        <rect>
         <x>250</x>
         <y>240</y>
         <width>16</width>
         <height>31</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="text">
        <string>x</string>
       </property>
      </widget>
     </widget>
     <widget class="QGraphicsView" name="GraphicsView_Pic">
      <property name="geometry">
       <rect>
        <x>540</x>
        <y>360</y>
        <width>451</width>
        <height>391</height>
       </rect>
      </property>
     </widget>
     <widget class="QGroupBox" name="groupBox_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>460</y>
        <width>511</width>
        <height>311</height>
       </rect>
      </property>
      <property name="title">
       <string>Deep Neural Network</string>
      </property>
     </widget>
     <widget class="QGroupBox" name="groupBox_3">
      <property name="geometry">
       <rect>
        <x>540</x>
        <y>20</y>
        <width>451</width>
        <height>331</height>
       </rect>
      </property>
      <property name="title">
       <string>Training Parameters</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="Tab_GAN">
     <attribute name="title">
      <string>ArtGAN</string>
     </attribute>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1385</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
